src/components/application_manager/src/app_service_manager.cc

```diff
@@ -195,6 +195,17 @@ std::vector<smart_objects::SmartObject> AppServiceManager::GetAllServices() {
   return services;
 }
 
+bool AppServiceManager::GetAppServiceInfo(std::string service_id,
```

Comment:
This is redundant, just use `FindServiceByID`

Reply:
Probably should wait to change it. You changed the return type from a std::pair to a pointer in app service manager lock PR